db/migrations/20190308111729_create_ilk_history_query.sql

```diff
@@ -1,30 +0,0 @@
--- +goose Up
--- +goose StatementBegin
-
--- Function returning the history of a given ilk as of the given block height
-CREATE FUNCTION api.all_ilk_states(ilk_identifier TEXT, block_height BIGINT DEFAULT api.max_block(),
```

Comment:
This function should be totally obsolete now, since its functionality can replicated by filtering the `allHistoricalIlkStates` function that postgraphile exposes